From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: sid <sidtosh4@gmail.com>
Date: Sat, 1 Apr 2023 21:13:19 +0000
Subject: [PATCH] libbrasero-burn: Fix incorrect condition during error check

If we don't have a GError set in priv->error, we report a generic
error message along with the process exit code, else we report the
actual GError message.
---
 libbrasero-burn/burn-process.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libbrasero-burn/burn-process.c b/libbrasero-burn/burn-process.c
index bd0b8d56f86c..cad1e8da53ff 100644
--- a/libbrasero-burn/burn-process.c
+++ b/libbrasero-burn/burn-process.c
@@ -251,7 +251,7 @@ brasero_process_finished (BraseroProcess *self)
 
 	/* check if an error went undetected */
 	if (priv->return_status) {
-		if (priv->error) {
+		if (!priv->error) {
 			brasero_job_error (BRASERO_JOB (self),
 					   g_error_new (BRASERO_BURN_ERROR,
 							BRASERO_BURN_ERROR_GENERAL,
